backend/glp/admin.py

```
@@ -0,0 +1,322 @@
+            return
+        for account in queryset:
+            try:
+                acknowledge_decrease(
+                    user=account.user, origin=origin, actor=request.user,
+                    note='admin action',
+                )
+                self.message_user(
+                    request, f'Acknowledged {origin} decrease for {account.user}.',
+                )
+            except GLPServiceError as error:
+                self.message_user(request, str(error), messages.ERROR)
```

Ruff (0.16.1)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)